csrc/device_lower/utils.cpp

```diff
@@ -575,7 +575,7 @@ class ReplaceExprInput : private kir::ExprMutator {
     auto replaced_inputs = getMaybeInputReplacementMap(node);
     if (replaced_inputs.has_value()) {
       auto replacement = IrBuilder::create<LoadStoreOp>(
-          node->opType(), node->out(), node->in());
+          node->opType(), node->out(), node->in(), node->cacheOp());
```

Comment:
@zasdfgbnm Isn't there a common API to create a copy of an expression?

Reply:
Are you referring to this? 
https://github.com/NVIDIA/Fuser/blob/50e994627d4bad5893cb40abe42a78523c78ef44/csrc/ir/base_nodes.h#L538
I recall at some point, I tried to clean up this part using that. There were some challenges about whether we want to replace the original expr, or we want to create a new expr on the side (such as for recompute). I didn't continue because that was not a priority for me.

Reply:
Thanks @zasdfgbnm.